devenv.nix

```diff
@@ -0,0 +1,16 @@
+{ pkgs, lib, ... }:
+
+{
+  # Packages
+  packages = [
+    pkgs.nodePackages.pnpm
+  ];
```

Comment:
Are additional packages required?

For me it was sufficient ot only add `pnpm`.